src/services/service-line-template/index.html

```diff
@@ -207,8 +207,21 @@ <h3 class="h4 text-700">{% t software-modernisation.card-4-heading %}</h3>
   </div>
 </section>
 
-<div class=soft-mod__sheen-container>
+{% assign featured_posts =  site.posts | where: 'pinned_locations', "testytest" %}
+{% if featured_posts.size == 1 %}
+  {% assign sheen_container_class = "soft-mod__sheen-container--one-publication" %}
+  {% assign curated_publications_class = "services__curated-publications--one-publication" %}
+{% else %}
+  {% assign sheen_container_class = "soft-mod__sheen-container" %}
+  {% assign curated_publications_class = "services__curated-publications" %}
+{% endif %}
```

Comment:
Let's revisit this logic around CSS classNames. It's becoming harder to follow, there might be a way we can simplify this

Reply:
This was fixed